string name_pickup
float32 xt
float32 yt
string name_deliver
float32 xg
float32 yg
